core/controllers/library.py

```diff
@@ -93,48 +108,78 @@ def get(self):
 class LibraryIndexHandler(base.BaseHandler):
     """Provides data for the default library index page."""
 
-    def get(self):
+    def get(self, **kwargs):
```

Comment:
No, this handler should be unchanged, I think. It's for the library index page. If we need separate JSON handlers for the other pages let's just make new JSON handlers that each have one task, otherwise there's going to be too much cyclomatic complexity in this one since it has to do so many different things.